assets/src/edit-story/components/canvas/editLayer.js

```diff
@@ -77,7 +77,8 @@ function EditLayerForElement({ element }) {
 
   const focusCanvas = useFocusCanvas();
 
-  useKeyDownEffect(ref, { key: 'esc', editable: true }, () => clearEditing(), [
+  const key = ['esc', 'tab'];
```

Comment:
Hmm. This definitely works for text, but does it work for image editor? E.g. we may want to tab between the pannable image and the scale slider?

Reply:
The issue comes straight from Fishfood. @pbakaus, what's your take on this? Would users want to tab in editmode for images and expect it to do other things than exit?

Reply:
I think the main question is whether there's another way to get to the scale slider using the keyboard...if not, then I think it makes sense to revert course and not modify this behavior (so we treat edit mode as a state that captures focus, and you always need esc to get out of it).

Reply:
My first question is why doesn't "focusout" doesn't work?